examples/eks-cluster-with-observability/README.md

```diff
@@ -0,0 +1,215 @@
+# EKS Cluster with Amazon Managed Grafana and Amazon OpenSearch Service
+
+This example demonstrates how to use the AWS EKS Accelerator for Terraform to deploy a new Amazon EKS Cluster with Prometheus for metrics and Fluent Bit for logging. It also provisions Amazon Managed Prometheus (AMP), Amazon OpenSearch Service within a VPC, and integrates AMP with Amazon Managed Grafana (AMG). It also deploys an EC2 instance to let us test OpenSearch. Lastly, it includes a sample workload, provisioned with ArgoCD, to generate logs and metrics.
```

Comment:
Lets call this a bastion host. Everywhere where you reference it, change the name to bastion.